File: mario.py
from cs50 import get_int

def func():
    while True:
        H = get_int("Height: ")
        W = H
        if H >= 0 and H <= 23:
            break

    for ixx in range(1, H + 1):
        no_of_hashes = ixx
        num_of_spaces = W - no_of_hashes

        print(" " * num_of_spaces, end="")
        print("#" * no_of_hashes, end="")

        print("  ", end="")

        print("#" * no_of_hashes)


if __name__ == "__main__":
    func()

File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n>8||n<1);

    for(int i=0;i<n;i++)
    {
       for(int d= n-1; d>i;d = d-1)
            {
                printf(" ");
            }
       for(int h=-1;h<i;h++)
        {

            printf("#");
        }
        printf("\n");
    }
}